core/container/src/main/java/org/wildfly/swarm/internal/SwarmMessages.java

```diff
@@ -162,6 +162,11 @@
     // ------------------------------------------------------------------------
 
     @LogMessage(level = Logger.Level.INFO)
+    @Message(id = 88888, value = "\n\n========================================================================\n\n%s\n\n========================================================================\n")
```

Comment:
Using `%n` instead of `\n` will use the platform-specific line separator as returned by System.getProperty("line.separator").

Reply:
If the Formatter class is used, otherwise ignore that

Reply:
Not explicitly using a formatter class.  No idea what is used internally.  @jamezp ?

Reply:
Internally it will end up using `String.format()` which uses `java.util.Formatter`.